Source/Scene/QuadtreePrimitive.js

```diff
@@ -997,7 +997,7 @@ define([
             error = error - CesiumMath.fog(distance, frameState.fog.density) * frameState.fog.sse;
         }
 
-        return error;
+        return error * frameState.screenSpaceErrorPixelRatio;
```

Comment:
I'm not seeing consistent results between this branch and master, even if this and below are changed to a `/`. I mainly just spot checked it by looking at the tile coordinates in Cesium Inspector.